proc_CPMG.py

```diff
@@ -18,52 +18,10 @@
         ]:
     s =  find_file(searchstr, exp_type=exp_type,
             expno=nodename, postproc=postproc, lookup=postproc_dict)
-    #{{{select and display coherence channel centered
-    best_shift = hermitian_function_test(s)
-    logger.info(strm("best shift is",best_shift))
-    s.setaxis('t2', lambda x: x-best_shift)
-    fl.next('time domain after hermitian test')
-    fl.image(s)
-    #}}}
     #{{{ centering CPMG echo
     s = center_CPMG_echo(s)
     fl.next('centered echo')
     fl.image(s)
-    #{{{cost function phase correction
-    s.ft('t2')
-    f_axis = s.fromaxis('t2')
-    def costfun(p):
-        zeroorder_rad,firstorder = p
-        phshift = exp(-1j*2*pi*f_axis*(firstorder*1e-6))
-        phshift *= exp(-1j*2*pi*zeroorder_rad)
-        corr_test = phshift * s
-        return (abs(corr_test.data.imag)**2)[:].sum()
-    iteration = 0
-    def print_fun(x, f, accepted):
-        global iteration
-        iteration += 1
-        logger.info(strm(iteration, x, f, int(accepted)))
-        return
-    sol = basinhopping(costfun, r_[0.,0.],
-            minimizer_kwargs={"method":'L-BFGS-B'},
-            callback=print_fun,
-            stepsize=100.,
-            niter=100,
-            T=1000.
-            )
-    zeroorder_rad, firstorder = sol.x
-    phshift = exp(-1j*2*pi*f_axis*(firstorder*1e-6))
-    phshift *= exp(-1j*2*pi*zeroorder_rad)
-    s *= phshift
-    logging.info(strm("RELATIVE PHASE SHIFT WAS %0.1f\\us and %0.1f$^\circ$", firstorder, angle(zeroorder_rad)/pi*180))
-    if s['tE',0].data[:].sum().real < 0:
-        s *= -1
-    logger.info(strm(ndshape(s)))
-    fl.next('after phased - real ft')
-    fl.image(s.real)
-    fl.next('after phased - imag ft')
-    fl.image(s.imag)
-    #}}}
```

Comment:
I think this is inferior to what we're going now, but I later salvage it anyways